test/aleph/classloader_test.clj

```diff
@@ -39,7 +41,7 @@
 (deftest test-classloader
   (testing "classloader: ensure the class loader is always a DynamicClassLoader"
     (let [result (CompletableFuture.)]
-      (with-dynamic-redefs [netty/operation-complete (partial operation-complete result)]
+      (with-dynamic-redefs [#_{:clj-kondo/ignore [:private-call]} netty/operation-complete (partial operation-complete result)]
```

Comment:
@KingMob Had to add this one after the rebase - not sure if the var being private is intended here?

Reply:
That's not something we want to expose externally, Aleph is already exposing way to many public `vars` in my opinion.
Having this `:private-call` is fine for me.

Reply:
Can that be set at another level? I’d prefer not to have kondo annotations deep in code if possible.

Reply:
@DerGuteMoritz Was there a way to set this elsewhere?

Reply:
Even better: Looks like adding https://github.com/clj-commons/aleph/pull/583/files#diff-d8bfac7f9aa64d7a63a2c6810095e91e716527266288a210b3f31a2c141ce3f2R3 made it completely unnecessary :smile: